scripts/setIsLastCommentAdmin.js

```diff
@@ -0,0 +1,35 @@
+const admin = require('firebase-admin')
+const { initFirebaseAdmin } = require('./utils/initFirebaseAdmin.js')
+
+initFirebaseAdmin()
+
+const setIsLastCommentAdmin = async () => {
```

Comment:
What do you think about adding a little bit of logging? This is a script and it'd be useful for the user of the script to know what's happening. Things like `Loading requests...`, `Setting "isLastCommentAdmin: (true/false)" to request id (id)...`, `Finished updating database`.